2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,21 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
-
+for (const key in friends){
```

Comment:
Watch your formatting. Are you using prettier?

This works, but there is not need to use keys. Why not try using filter()?